### DISCUSSION TITLE
[v4] `before:absolute` class isn't generated

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [tailwindlabs/tailwindcss#15134] [Help]
I'm using the latest versions of `tailwindcss` and `@tailwindcss/vite` on a SvelteKit project, where I have this element:
```svelte
<Card.Root
	id={word.id}
	class="
		relative flex-1 w-auto flex justify-between border-2 transition-colors
		before:absolute before:inset-y-0 before:w-1 before:rounded-s-md before:bg-(--category-color)
		before:transition-[width] has-[a:hover]:border-(--category-color) has-[a:hover]:before:w-2
	"
	style="--category-color: var(--color-category-{word.usage_category})"
>
```

However, upon inspecting, the classes `before:transition-[width]` and `before:absolute` are simply not included in the generated CSS.

**Comment by adamwathan** [comment]
Hey! Can you share a reproduction? I can't reproduce this locally and can't think of a reason why that class specifically would be ignored 🤔

**Reply by TheOnlyTails** [reply]
Hey, I tried reproducing this in the playground but couldn't do it either, I can provide the source code for the project and a deployed example:
https://font-submission.lipu-svelte.pages.dev/
the offending line: https://github.com/lipu-linku/lipu/blob/font-submission/src/routes/Entry.svelte#L42
config: https://github.com/lipu-linku/lipu/blob/font-submission/src/app.css
